Packs/Logzio/TestPlaybooks/playbook-Logzio-Test.yml

```diff
@@ -0,0 +1,83 @@
+id: Logzio-Test
+version: -1
+name: Logzio-Test
+starttaskid: '0'
+tasks:
+  '0':
+    id: '0'
+    taskid: 705937ce-ece2-40a5-8c07-b0a277223762
+    type: start
+    task:
+      id: 705937ce-ece2-40a5-8c07-b0a277223762
+      version: -1
+      name: ''
+      iscommand: false
+      brand: ''
+      description: ''
+    nexttasks:
+      '#none#':
+      - '1'
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 450,\n    \"y\": 50\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  '1':
+    id: '1'
+    taskid: 733ec9dd-e04a-40d1-87c9-27d310752d66
+    type: regular
+    task:
+      id: 733ec9dd-e04a-40d1-87c9-27d310752d66
+      version: -1
+      name: logzio-get-logs-by-rule-id
+      description: Returns the raw logs that triggered the security rule in Logz.io.
+      script: Logz.io|||logzio-get-logs-by-rule-id
+      type: regular
+      iscommand: true
+      brand: Logz.io
+    nexttasks:
+      '#none#':
+      - '2'
+    scriptarguments:
+      id:
+        simple: ${incident.labels.alertEventId}
+      size:
+        simple: '100'
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 450,\n    \"y\": 230\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  '2':
+    id: '2'
+    taskid: b3dd558c-ccc2-4d65-8c29-f91734560b23
+    type: regular
+    task:
+      id: b3dd558c-ccc2-4d65-8c29-f91734560b23
+      version: -1
+      name: logzio-search-logs
```

Comment:
please verify both commands outputs in this test

Reply:
Both commands output fixed
there is a new playbook yml file